.github/ISSUE_TEMPLATE/bug_report.yaml

```
@@ -0,0 +1,123 @@
+name: Bug report
+description: Create a report to help us improve
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! Please make sure to fill out the entire form below, providing as much context as you can in order to help us triage and track down your bug as quickly as possible.
+
+        Before filing a bug, please be sure you have searched through [existing bugs](https://github.com/open-telemetry/opentelemetry-collector/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Abug) to see if an existing issue covers your bug.
+  - type: dropdown
+    id: component
+    attributes:
+      label: Component(s)
+      description: Which component(s) does your bug report concern?
+      multiple: true
+      options:
+      # NOTE: The list below is autogenerated using `make generate-gh-issue-templates`
+      # Do not manually edit it.
+      # Start Collector components list
+      - cmd/mdatagen
+      - cmd/mdatagen/internal/sampleprocessor
+      - cmd/mdatagen/internal/samplereceiver
+      - cmd/mdatagen/internal/samplescraper
+      - confmap
+      - connector/forward
+      - connector/x
```

This should be `connector/xconnector`, right?
```suggestion
      - connector/xconnector
```

---

this is kind of a limitation of the githubgen tool currently. I didn't get around to fix it yet, but I think for now it should be fine. I don't think those components are selected often if at all in the issue templates.